stdlib/2and3/distutils/version.pyi

```diff
@@ -0,0 +1,35 @@
+# Stubs for distutils.version (Python 2 and 3.5)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+import sys
+from typing import Any
+
+class Version:
+    def __init__(self, vstring=None): ...
+
+    if sys.version_info >= (3,):
```

Comment:
It probably doesn't matter that the Python 3 version has them, since the return values are identical to the ones on `object`. (And that the implementation differs is irrelevant for typing)